### DISCUSSION TITLE
Format date?

### DISCUSSION THREAD

**Discussion by paulywill** [AbhyudayaSharma/react-git-info#21] [Q&A] [answered]
Is there a way to pass a formatting argument?

i.e. `--date=format:'%Y-%m-%d %H:%M:%S'`

**Comment by AbhyudayaSharma** [comment, marked as answer]
Hi @paulywill. The date is returned as a strict [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) string and can be passed to a `Date` constructor. You can then use [`Intl.DateTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) with that date to format it to your liking.

```ts
const date = new Date(gitInfo.commit.date);
const options = {
  year: 'numeric', month: 'numeric', day: 'numeric',
  hour: 'numeric', minute: 'numeric', second: 'numeric',
  hour12: false,
  timeZone: 'America/Los_Angeles'
};
console.log(new Intl.DateTimeFormat('en-US', options).format(date))
```
